src/org/company/MySuperLongClass.java

```diff
@@ -0,0 +1,18 @@
+package org.company;
+
+import org.company.abstracts.AbstractMyClass;
+
+/**
+ * @since Current.Project.Version
+ */
+public class MySuperLongClass {
+
+    public MySuperLongClass() {
+        //
+    }
+
+    public MySuperLongClass(final int one, final String two, final Object three, final long four, final double five, final float six, final MyEnum seven, final MyClass eight, final AbstractMyClass nine) {
```

Comment:
This need to be split up. Its my preference that if we are going to split it up, that every parameter gets its own line.